content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -0,0 +1,573 @@
+---
+title: Install the Datadog Agent with Embedded OpenTelemetry Collector
+private: true
+further_reading:
+- link: "/opentelemetry/agent/agent_with_custom_components"
+  tag: "Documentation"
+  text: "Use Custom OpenTelemetry Components with Datadog Agent"
+---
+
+## Overview
+
+The Datadog Agent with embedded OpenTelemetry (OTel) Collector is an open source Collector distribution that includes:
+
+- Built-in Datadog pipelines and extensions
+- Support for traces, metrics, and logs
+- A curated set of components for optimal performance with Datadog
+
+{{< img src="/opentelemetry/embedded_collector/architecture.png" alt="Architecture overview for Collector embedded in the Datadog Agent." style="width:100%;" >}}
+
+Follow this guide to install the Datadog Agent with the OpenTelemetry Collector using Helm.
+
+<div class="alert alert-info">If your application is already instrumented with the OTel SDK or if you want to build a Datadog Agent image with additional OpenTelemetry components, read <a href="/opentelemetry/agent/agent_with_custom_components">Use Custom OpenTelemetry Components with Datadog Agent</a>.<br>For a list of components included by default, see <a href="#included-components">Included components</a>.</div>
+
+
+## Requirements
+
+To complete this guide, you need the following:
+
+**Datadog account**:
+1. [Create a Datadog account][1] if you don't have one.
+1. Find or create your [Datadog API key][2].
+1. Find or create your [Datadog application key][3].
+
+**Software**:
+Install and set up the following on your machine:
+
+- A Kubernetes cluster (v1.29+)
+- Helm (v3+)
+- [Docker][50]
+- [kubectl][5]
+- Ensure all OTel components are compatible with the [embedded OpenTelemetry Collector version][51]
```

Comment:
The idea here is that the OTel components are compatible with the otelcol_version, for example 0.104.0 right now. Does that make sense? I suppose it might make more sense in the BYOC doc actually.

Reply:
Yes. remove from here